src/main/java/net/pms/encoders/MEncoderVideo.java

```diff
@@ -251,10 +251,10 @@ public void itemStateChanged(ItemEvent e) {
 					fakemedia.setCodecV("mpeg4");
 					fakemedia.setContainer("matroska");
 					fakemedia.setDuration(45d*60);
-					audio.getAudioProperties().setNumberOfChannels(2);
+					audio.setNumberOfChannels(2);
 					fakemedia.setWidth(1280);
 					fakemedia.setHeight(720);
-					audio.setSampleFrequency("48000");
+					audio.setSampleRate(48000);
```

Comment:
Thanks, i did found the fact to have the two naming in the code was confusing, i needed trying to remember or search to check if it was the same value.